inlang/source-code/versioned-interfaces/project-settings/src/interface.ts

```diff
@@ -76,6 +76,9 @@ const InternalProjectSettings = Type.Object({
 			],
 		})
 	),
+	featureFlags: Type.Optional(
+		Type.Record(Type.String(), Type.Union([Type.Boolean(), Type.Number(), Type.String()]))
+	),
```

Comment:
@samuelstroschein what do you think about adding featureFlags to project settings?
This would simplify testing of projects with/without a feature flag
We could still have logic to feature-flag by owner/repo in addition to this?